src/components/lists/items/CallAssignmentListItem.jsx

```diff
@@ -41,48 +44,104 @@ export default class CallAssignmentListItem extends React.Component {
 
     render() {
         let assignment = this.props.data;
+        let progressSum = 0;
+        let callsStats = null;
+        let reachedStats = null;
         let targetStats = null;
         let goalStats = null;
-        let participantList = null;
+        let participantIndicator = null;
+        const assignmentDateStart = new Date(assignment.start_date);
+        const assignmentDateEnd = new Date(assignment.end_date);
+        const inPast = (assignmentDateEnd < (new Date()) ? true : false);
+
+        const classNames = cx({
+            'CallAssignmentListItem': true,
+            'past': inPast
+        });
+
+        const assignmentDateSpan =
+            assignmentDateStart.getDate()
+            + "/" + assignmentDateStart.getMonth()
```

Comment:
`Date.getMonth()` returns months zero-indexed, i.e. january is 0, february is 1, december is 11. So this will yield a string that is not what a human would expect.